examples/deepspeed-multinode/sky.yaml

```
@@ -0,0 +1,80 @@
+# Example: a distributed DeepSpeed job (DeepSpeed-Chat) on 2 VMs.
+#
+# This takes care constructing a "hostfile" to pass to DeepSpeed.
+#
+# Usage:
+#
+#   $ sky launch sky.yaml -r --down -c ds
+#
+#   # After the job starts running, log into the two nodes and check gpustat:
+#
+#   $ ssh ds
+#   $ gpustat -i
+#
+#   $ ssh ds-worker1
+#   $ gpustat -i
+
+resources:
+  accelerators: T4:1  # AWS, Azure, GCP
+  # accelerators: A100:1  # GCP, Lambda
+  # accelerators: A100-80GB:1  # Azure, GCP, SCP
+num_nodes: 2
+
+setup: |
+  set -ex
+  git clone https://github.com/microsoft/DeepSpeedExamples.git || true
+  cd DeepSpeedExamples
+  git checkout d7c42b4f34df91035e7ed3e0c51500bb53d0bc71
+
+  conda create -n deepspeed python=3.8 -y
+  source activate deepspeed
```

nit: we use `conda activate` everywhere else.
```suggestion
  conda activate deepspeed
```